Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -473,18 +473,25 @@ public ObservableList<VariableBase> GetBFandActivitiesVariabeles(bool includePar
                     {
                         var.ParentType = GingerDicser.GetTermResValue(eTermResKey.Activity);
                         var.ParentGuid = activ.Guid;
-                        var.ParentName = activ.ActivityName;
+                        if (string.IsNullOrEmpty(activ.ActivitiesGroupID))
+                        {
+                            var.ParentName = string.Format("{0}\\{1}", this.Name, activ.ActivityName);
+                        }
+                        else
+                        {
+                            var.ParentName = string.Format("{0}\\{1}\\{2}", this.Name, activ.ActivitiesGroupID, activ.ActivityName);
+                        }
+                    }
+                    if (includeOnlyPublishedVars && var.Publish == false)
+                    {
+                        continue;
                     }
-                    if (includeOnlySetAsInputValue)
+                    if (includeOnlySetAsInputValue && var.SetAsInputValue == false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4903551)